bitbots_vision/src/bitbots_vision/vision.py

```diff
@@ -90,7 +90,7 @@ def _image_callback(self, image_msg):
         """
         # drops old images and cleans up queue
         image_age = rospy.get_rostime() - image_msg.header.stamp
-        if image_age.to_sec() > 1.0:
+        if 1.0 < image_age.to_sec() < 1000.0:
```

Comment:
Why? is this for ROS bags? please add a comment.

Reply:
Yes its for ros bags. And i added an comment.